desktop/apps/auction/routes.coffee

```diff
@@ -48,7 +48,6 @@ setupUser = (user, auction) ->
 
   artworks.comparator = (artwork) ->
     saleArtwork = artwork.related().saleArtwork
-    saleArtwork.get('lot_number') or
```

Comment:
We should be sorting by `position` first, especially now that `lot_label` is a string.